sormas-backend/src/test/resources/performance/sql_log.sh

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Move this file to sormas-backend/utils and give it a better name, for example `watch_sormas_db_log.sh`